### PR TITLE
extra command for trimming video and volume up, added -preset ultrafast flag to uploadable version command

### PR DESCRIPTION
**Pull request by igauravsehrawat**

### CODE REVIEW

**Comment by shrayasr** [comment]
Apart from that, LGTM :+1: :shipit:

File: README.md [hunk 1]
```diff
@@ -45,7 +45,11 @@ This repository contains helper scripts and commands to create meetup videos for
 
 * Trim the audio if required
 
-        $ sox cleaned.wav final.wav trim 00:02 =21:20 
+        $ sox cleaned.wav final.wav trim 00:02 =21:20
+
+* Trim the video if required
+
+        $ ffmpeg -i <input_video_name> -ss <start_time_in_hh:mm:sec> -to <end_time_in_hh:mm:sec> <output_video_name>
```
Comment (argvk) [inline README.md]: I think you might want to move `-ss` before `-i`

FWIW, prepending `ss` seeks to keyframe (faster) vs appending `ss` decodes and discards until that time is reached (slower)

checkout : https://trac.ffmpeg.org/wiki/Seeking
this is also used here ->  https://github.com/argvk/chennaipy-video-helpers/blob/413a98ffc9464ae9525e74f333825e8c35064c3e/create_video_segments.py#L37-L52

File: README.md [hunk 2]
```diff
@@ -59,16 +63,24 @@ This repository contains helper scripts and commands to create meetup videos for
...
+        $ ffmpeg -i <input_video_name> -af 'volume=2.5<or 1x>' -preset ultrafast <output_video_name>
+
+* scale video to 16:9
 
         $ ffmpeg -i final.mp4 -vf pad="ih*16/9:ih:(ow-iw)/2:(oh-ih)/2" -preset ultrafast -acodec copy final2.mp4
 
-* Generate the title image 
+    or specify absolute pixels
+
+        $ ffmpeg -i <input_video_name> -vf "scale=1024:768" -preset ultrafast <output_video_name>
```
Comment (argvk) [inline README.md]: why the scaling ? this is not the native aspect ration to be uploaded too. 16:9 is the preferred aspect ratio for the final video.

File: README.md [hunk 3]
```diff
@@ -77,4 +89,4 @@ This repository contains helper scripts and commands to create meetup videos for
 
 * create the uploadable version
 
-        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]" uploadable.mp4
+        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]" -preset ultrafast uploadable.mp4
```
Comment (argvk) [inline README.md]: I would make this optional (probably in a fineprint), wouldn't mind if this takes longer since its the last step, otherwise the final videos tend to get huge and become a pain to share around before uploading to vimeo/youtube.

**Comment by argvk** [comment]
Thanks for the PR @igauravsehrawat, I've left a few comments

File: README.md (hunk 3 above)
Comment (shrayasr) [inline README.md]: :+1: agree

File: README.md (hunk 2 above)
Comment (igauravsehrawat) [inline README.md]: yes, you are correct but i was getting an error: Input area 0:0:1366:768 not within the padded area 0:0:1364:768 or zero-sized since i recorded the screen on my laptop.
so thought it would be helpful to others.

File: README.md (hunk 3 above)
Comment (igauravsehrawat) [inline README.md]: issue wasn't about long time, without -preset ultrafast it was using all four cores to 100 %, if load sharing could be used that would useful somehow. 
with ultrafast the cpu cores were around 60-65% .

File: README.md (hunk 1 above)
Comment (igauravsehrawat) [inline README.md]: ok. added in new commit .

File: README.md (hunk 3 above)
Comment (shrayasr) [inline README.md]: Ultrafast probably is making a wager to bring the cores down to that percent. Not using it will take time and more resources but it'll bring down the final size of the video. 

-1 for having the `-present ultraflag` in here from me

Comment (igauravsehrawat) [inline README.md]: ok. seconded your thought. added a note for `ultrafast' flag.

File: README.md [hunk 4]
```diff
@@ -59,16 +63,24 @@ This repository contains helper scripts and commands to create meetup videos for
...
+
+* scale video to 16:9
 
         $ ffmpeg -i final.mp4 -vf pad="ih*16/9:ih:(ow-iw)/2:(oh-ih)/2" -preset ultrafast -acodec copy final2.mp4
 
-* Generate the title image 
+    or specify absolute pixels
+
+        $ ffmpeg -i <input_video_name> -vf "scale=<specify_width_in_px:specify_length_in_px>" -preset ultrafast <output_video_name>
+
```
Comment (argvk) [inline README.md:77]: I'm still don't know why the scaling is required, what are trying to solve/introduce here ? its not intuitive to enter the HxW resolution, moreover this doesn't care about maintaining the aspect ration. 

but my primary question remains the same, what is the need for this. 

the rest of the change look good, I'll go ahead with those, I want to hold back on this.

File: README.md [hunk 5]
```diff
@@ -77,4 +89,6 @@ This repository contains helper scripts and commands to create meetup videos for
 
 * create the uploadable version
 
-        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]" uploadable.mp4
+        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]"  uploadable.mp4
+
+    Note: if you aren't worried about file size & want to avoid 100% all cpu core utilization, you can add `-preset ultrafast` to speed up the processing
```
Comment (argvk) [inline README.md:94]: I'll slightly reword this to:

> Use `-preset ultrafast` if cpu is consistently high, but increases overall filesize. Using it is **not** recommended. 

Let me know if its ok.

**Comment by argvk** [comment]
I've merge it https://github.com/argvk/chennaipy-video-helpers/commit/02bfa171f1c3434c801415d306a9ab993d2a7e35

thanks @igauravsehrawat, awesome work!

**Comment by shrayasr** [comment]
Awesome work @igauravsehrawat! :+1:

File: README.md (hunk 5 above)
Comment (igauravsehrawat) [inline README.md:94]: yeah, sure. I just wanted to highlight things which bothered me. I am absolutely fine with improvisation.

File: README.md (hunk 4 above)
Comment (igauravsehrawat) [inline README.md:77]: yes, you are correct but i was getting an error: Input area 0:0:1366:768 not within the padded area 0:0:1364:768 or zero-sized since i recorded the screen on my laptop.
so thought it would be helpful to others, if they get similar kind of error.
